### ISSUE TITLE
Debugging Script running in external application in VS Code

### ISSUE THREAD

**Issue by william-c-nc** [filipw/dotnet-script#615]
Hello,

First, I wanted to say that I am really liking the dotnet-script tool so far and see a lot of use cases for it 👍. 

I am working on an application that comes with an API that also supports running user defined scripts by reading the text file and creating a Roslyn CSharp script. I am looking to allow the user to debug their scripts via VS code, so I was wondering if it would be possible to compile and execute the script from the main application by calling dotnet-script, and then attach the VS code debugger somehow? 

I was thinking this could be done by running "dotnet-script exec" on the compiled .dll in the main application via Process.Start(...), but it was unable to actually execute any API methods (even though I have intellisense for them in vs code from the #r directive). I also tried a script that waits for a debugger to attach and calling dotnet-script on the script file via Process.Start(...), and then attaching a debugger in VS code, but I encountered the same issue. I am wondering if this is a limitation with dotnet-script, or perhaps with my API and setup.

Basically, before I go any further I want to see if this is possible in principle. Thanks!
